src/components/AutocompleteSelectMenu/AutocompleteSelectMenu.tsx

```diff
@@ -171,10 +169,10 @@ const AutocompleteSelectMenu: React.FC<AutocompleteSelectMenuProps> = props => {
                         : options
                       ).map((suggestion, index) => (
                         <MenuItem
-                          data-test-id={testIds[index]}
-                          key={suggestion.value}
+                          data-test-id={testIds?.[index] ?? ""}
```

Comment:
I don't see why we can't have an data-test-id available, is the `?` really necessary?

Reply:
`testIds` are optional